script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt

```diff
@@ -318,7 +318,7 @@ All of them―――
 -- Page 263, Offset 39128.
 -- 今の姫君に腕を振るわせるとは、称賛にあまりある”
 %{i}
-She deserves no faint praise for managing to display such strength when up against the current Princess."
+She deserves no faint praise for managing to display such strength when up against the current princess."
```

Comment:
Capitalizing Princess may be a Roa-ism, check with Petrikow

Reply:
Is that so? I dont think we were consistent with it in Arc route. Might as well ask him though. @Petrikow 
![image](https://user-images.githubusercontent.com/90473809/236686940-d7b6d58f-0eed-4205-9f75-40e80238a471.png)